agent/router/manager.go

```diff
@@ -235,6 +235,10 @@ func (m *Manager) FindServer() *metadata.Server {
 }
 
 func (m *Manager) checkServers(fn func(srv *metadata.Server) bool) bool {
+	if m == nil {
```

Comment:
This is presumably only done for belt-and-suspenders reasons right? Unless there's another lurking bug you don't expect this to trigger.

Reply:
Right. This should never happen buts it’s safe to ignore if somehow it ever did again.